wolfcrypt/src/random.c

```diff
@@ -3744,15 +3766,23 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
 
     #ifndef NO_DEV_URANDOM /* way to disable use of /dev/urandom */
         os->fd = open("/dev/urandom", O_RDONLY);
+        #if defined(DEBUG_WOLFSSL)
+            WOLFSSL_MSG_EX("open(/dev/urandom... %d", 0);
+        #endif
         if (os->fd == -1)
     #endif
         {
+    #if defined(DEBUG_WOLFSSL)
+            WOLFSSL_MSG_EX("open(/dev/urandom... %d", 1);
```

Comment:
No point in using %d
Do you mean /dev/random?

Reply:
yes, my bad. fixed.